app/models/user_emails.py

```diff
@@ -0,0 +1,23 @@
+from app.models import db
+
+
+class UserEmails(db.Model):
+    """email notifications model class"""
+    __tablename__ = 'user_emails'
+    id = db.Column(db.Integer,
+                   primary_key=True)
+    email = db.Column(db.String(120), unique=True, nullable=True)
```

Comment:
What will be the whole point of storing if this can be nullable 😄 ?

Reply:
It would be nice to remove field email from User model. I don't think that we need two fields which have the same responsibility.

Reply:
done @rafalkowalski and @poush 
Not able to migrate as other parts are using previous way to fetch email attr of user.
Let me know if I can proceed and also about
> should we have a different model for EmailTypes like personal, work, etc? or a it can be simply a custom field and user can provide anything?